editor.planx.uk/src/pages/FlowEditor/lib/analytics/provider.tsx

```diff
@@ -360,6 +361,39 @@ export const AnalyticsProvider: React.FC<{ children: React.ReactNode }> = ({
     });
   }
 
+  async function updateAutoAnsweredNodeWithAllowListAnswers(
+    nodeId: string,
+    breadcrumb: Store.UserData,
+  ) {
+    if (shouldSkipTracking()) return;
+
+    // Only proceed if it's a SetValue component that is appending or replacing (not removing) a value
+    //   Do not track allow list answers for Questions or Checklists because these will always be equal or less granular data values than put to the user
+    if (
+      flow[nodeId]?.type !== TYPES.SetValue ||
+      ["removeOne", "removeAll"].includes(flow[nodeId]?.data?.operation)
```

Comment:
It seems unlikely, but this could fail if we add more operations to SetValue.

We might be better checking if it's not `"replace"` or `"append"`.